_posts/2019-04-16-festivalsommer-2019.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: post
+title:  Festivalsommer 2019
+author: Louis
+date:   2019-04-18 12:30:00 +0200
+---
+In den letzen Jahren haben wir auf mehreren Veranstaltungen Freifunk aufgebaut, manchmal recht kurzfristig.
+Mit dabei war die [Breminale](/blog/2018/12/09/rueckblick-auf-die-breminale-2018.html), das Summersounds, das [Sommerfest Vorstraße](/blog/2018/06/17/sommerfest-vorstraße.html), das HeVie und das [13°-Festival](/blog/2018/11/06/13grad.html) in Bremen Nord. 
+Dazu nutzen wir meist unsere Outdoor-Boxen und Archer C7 Router von TP-Link.
+
+In diesem Jahr haben wir wieder vor, auf Veranstaltungen unterwegs zu sein.
+Solche Festivals sind neben dem Werbeeffekt eine gute Gelegenheit für uns, unsere Software mal unter Vollauslastung zu testen.
+Gleichzeitg lernt man beim Netzaufbau gegenseitig von einander und hat in der Gruppe echt viel Spaß.
+
+## Die Crew
+Damit das klappt brauchen wir Freiwillige, die mithelfen.
+Auch wenn man nicht so viel Erfahrung hat lernt man spätestens vor Ort viel dazu.
+
+Für unser größtes bespieltes Festiaval, die Breminale, beginnen wir jetzt schon mit den Planungen.
+Wenn du dir vorstellen kannst mitzuhelfen, kannst du dich gerne in unserer [Planungliste](https://dudle.hackerspace-bremen.de/FFHB_Breminale_2019/) eintragen.
```

Comment:
`, trage dich doch bitte in unsere Planungsliste ein.`

zwei mal direkt _kannst_ hintereinander ist auch nicht schön.